### DISCUSSION TITLE
Bulma Framework

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#1321] [Perguntas e Respostas]
Fala, galera!

alguém já utilizou ou utiliza o framework **[Bulma](https://bulma.io/)** em algum projeto? Gostaram dele? Odiaram? Veem alguma vantagem em relação a outros frameworks como Bootstrap, Materialize, etc?

Aproveitando o embalo, conhecem algum framework CSS pouco conhecido/citado, mas que recomendam dar uma olhada?

**Comment by johanguse** [comment]
Bulma é super leve... tá tendo muitos adeptos, espero usar logo em um projeto pessoal.
Da uma olhada nessa lista: https://github.com/troxler/awesome-css-frameworks

**Comment by tassoevan** [comment]
Eu gosto **muito** do Bulma, mas o visual dele é um pouco menos versátil sem customizações. Em desktop, por exemplo, você pode querer os seus componentes de UI "mais condensados" enquanto a configuração default do Bulma usa e abusa de margens e espaçamentos; precisa existir um balanceamento baseado na complexidade da interface.
